scripts/rancher_pr_deploy.sh

```diff
@@ -24,13 +24,15 @@ if [[ -n "$TRAVIS" ]]; then
     curl -s https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/c2s-pharmacy-finder/0/docker-compose.yml  -o docker-compose.yml
     curl -s https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/c2s-pharmacy-finder/0/rancher-compose.yml -o rancher-compose.yml
 
+    LATEST_RELEASE=$(curl -s https://api.github.com/repos/nhsuk/nearby-services-api/releases/latest | jq -r '.tag_name')
```

Comment:
Options for creating release and images:

- Script the creation of a release so we can also generate the tagged image at the same time. See [here](https://developer.github.com/v3/repos/releases/#create-a-release) for more details. Note: we should mark it as pre-release until we are ready to start using it.
- Alternative would be to use the GitHub release webhook

In both cases, we need somewhere to run the script/webhook. (?Azure Functions vs Docker Container)